the-deep/server
# Add Entry.verified status

Related to [#932](https://github.com/the-deep/deeper/issues/932)

To verify or unverify an entry
`POST at /api/v1/entries/__EntryId__/verify/ to verify`
`POST at /api/v1/entries/__EntryId__/unverify/ to unverify`

To filter by verification status
`GET` at `/api/v1/entries/` with request parameters `?verified=True` or `?verified=False`
